entries/serializers.py

```diff
@@ -65,6 +65,7 @@ def create(self, validated_data):
         session_answers = []
 
         # create the session answers
+        # create empty retention string if it doesn't exist
```

Comment:
the `if` below this is for when previous sessions exist - why should this be here?